netcontrold/tests/lib/test_util.py

```diff
@@ -19,8 +19,18 @@
 from netcontrold.lib import util
 
 _BASIC_CPU_INFO = """
```

Comment:
let it be _BASIC_CPU_INFO_0, the following one be _BASIC_CPU_INFO_1 ? .. just for the consistency in naming variables/test_functions as well..